worldedit-core/build.gradle

```diff
@@ -70,37 +70,39 @@ processResources {
     }
 }
 
-jar.archiveName="fawe-api-${project.parent.version}.jar"
```

Comment:
If you want to update gradle, update it properly by updating deprecated methods too.

Reply:
I'm looking into it as well tomorrow. The rest regarding Gradle follows the same comment.

Reply:
👍 
I'll just annotate it once.